cmd/util/ledger/migrations/cadence_values_migration.go

```diff
@@ -133,6 +133,11 @@ func (m *CadenceBaseMigrator) MigrateAccount(
 			}
 		}
 
+		// Load storage map.
+		for _, domain := range domains {
```

Comment:
Hard to see: What is `domains` here?

Reply:
`domains` are used for atree migration:
- https://github.com/onflow/flow-go/blob/master/cmd/util/ledger/migrations/atree_register_migration.go#L429-L437

```Go
var domains = []string{
	common.PathDomainStorage.Identifier(),
	common.PathDomainPrivate.Identifier(),
	common.PathDomainPublic.Identifier(),
	runtime.StorageDomainContract,
	stdlib.InboxStorageDomain,
	stdlib.CapabilityControllerStorageDomain,
}
```

Reply:
Perfect!